spyder/widgets/ipythonconsole/shell.py

```diff
@@ -84,6 +88,19 @@ def set_cwd(self, dirname):
             self.kernel_client.input(u'!' + code)
         else:
             self.silent_execute(code)
+        self._cwd = dirname
+
+    def update_cwd(self):
```

Comment:
The idea is to have the same name on both sides, i.e. frontend and kernel. Since we already have `get_cwd` in `SpyderKernel`, this should have the same as well.

So please change it to `get_cwd`.